peekingduck/runner.py

```diff
@@ -29,11 +29,14 @@ class Runner():
     """Runner class that uses the declared nodes to create pipeline to run inference
     """
 
-    def __init__(self, RUN_PATH: str, CUSTOM_NODE_PARENT_FOLDER: str = None,
+    def __init__(self, RUN_PATH: str,
+                 config_updates_cli: str,
+                 CUSTOM_NODE_PARENT_FOLDER: str = None,
                  nodes: List[AbstractNode] = None):
         """
         Args:
             RUN_PATH (str): path to yaml file of node pipeine declaration.
+            config_updates_cli (str): strings defining node names and configs to update.
```

Comment:
can we place the expected format here? `strings` is too vague

Reply:
updated